src/browser/api_protocol/api_handlers/application.js

```diff
@@ -361,10 +365,29 @@ function runApplication(identity, message, ack, nack) {
     if (manifestUrl) {
         addRemoteSubscription(remoteSubscription).then((unSubscribe) => {
             remoteSubscriptionUnSubscribe = unSubscribe;
-            Application.runWithRVM(identity, manifestUrl).catch(nack);
+            unsub = duplicateUuidTransport.subscribeToUuid(uuid, (e) => {
```

Comment:
Is this assignment a race against lines 357 above in any way? They are both async no?

Reply:
This will always happen before line 357, but I have the typeof check above to be sure

Reply:
👍